README.md

```diff
@@ -27,6 +27,7 @@ Before working with the Jekyll code, you will need to install Jekyll:
     $ gem install jekyll
     $ gem install jekyll-redirect-from
     $ gem install html-proofer
+    $ gem install activesupport -v 4.2.7
```

Comment:
See https://github.com/apache/incubator-beam-site/pull/50 

Hopefully that solves the issue so this PR is unnecessary?